DataFormats/Detectors/FIT/FT0/include/DataFormatsFT0/RawEventData.h

```diff
@@ -0,0 +1,95 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+//
+//file RawEventData.h class  for RAW data format
+// Artur.Furs
+//
+#ifndef ALICEO2_FIT_RAWEVENTDATA_H_
+#define ALICEO2_FIT_RAWEVENTDATA_H_
+
+#include "DataFormatsFT0/Digit.h"
+#include "Headers/RAWDataHeader.h"
+#include "DataFormatsFT0/LookUpTable.h"
+#include <CommonDataFormat/InteractionRecord.h>
+#include <iostream>
+#include <bitset>
+#include "Rtypes.h"
+namespace o2
+{
+namespace ft0
+{
+constexpr int NCHANNELS_FT0 = 208;
+constexpr int NCHANNELS_PM = 12;
+constexpr int NPMs = 18;
+constexpr int NBITS_EVENTDATA = 9;
+
+struct EventHeader {
+  uint startDescriptor : 4;
+  uint Nchannels : 4;
+  uint reservedField : 32;
+  uint orbit : 32;
+  uint bc : 12;
+  ClassDefNV(EventHeader, 1);
+};
+struct EventData {
+  int time : 12;
```

Comment:
Actually, after rechecking I see that it is much worse: your EventData is in fact 64 bits long (due to the alignment to int), out of which only 1st 37 bits are used. Then, since neither of its members exceeds 16 bits, you could use ``short`` instead of the ``int`` (for ``charge`` and ``channelID``, for the rest just use bool), this will reduce its size to 48 bytes.
See next comment